examples/tts/conf/fastpitch_align_44100.yaml

```diff
@@ -0,0 +1,186 @@
+name: "FastPitch"
+sample_rate: 44100
+train_dataset: ???
+validation_datasets: ???
+prior_folder: ???
+
+model:
+  learn_alignment: true
+  n_speakers: 1
+  symbols_embedding_dim: 384
+  max_token_duration: 75
+  n_mel_channels: 80
+  pitch_embedding_kernel_size: 3
+  n_window_size: 2048
+  n_window_stride: 512
+  fmax: null
+  pitch_fmin: 80
+  pitch_fmax: 640
+  pitch_avg: 211.27540199742586
+  pitch_std: 52.1851002822779
```

Comment:
Can you add a comment what pitch statistics these values are for?

Reply:
Added